assets/sass/modules/_forms.scss

```diff
@@ -189,6 +189,40 @@
       right: 10px;
       opacity: 0.5;
     }
+
+    .password-toggle {
+
+      .button-off {
+        display: none;
+      }
+
+      .button {
+        right: 5px;
+        position: absolute;
+        top: 50%;
+        margin-top: -13px;
+        line-height: 24px;
+        height: 26px;
+        padding: 0 5px;
+        font-size: 11px;
+      }
+
+      .button-on {
+        background: #929292;
```

Comment:
Theming (the colors we use amongst other things) should go into okta-theme.scss, and we generally use variables to capture these types of hardcoded colors (i.e. in _variables.scss). @Ujjwalreddy-okta probably can add a more helpful note here.